serverTest.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+import server
+
+class TestSocketIO(unittest.TestCase):

```

Comment:
Take a read at https://defragdev.com/blog/?p=783
And think about what we've discussed offline as well.